apps/challenges/urls.py

```diff
@@ -201,4 +201,9 @@
         views.get_all_challenge_templates,
         name="get_all_challenge_templates",
     ),
+    url(
```

Comment:
@Ram81 Is it necessary to have a new API? Can't we use the previous API with the number of chunks parameter?

Reply:
@RishabhJain2018 the new API is to complete the chunked upload. The flow is 
1. Generate multipart upload id using `create_multipart_upload`
2. Generate `num_chunks` presigned_urls for `upload_id` returned by `create_multipart_upload` API
3. Upload chunks to the new presigned urls and store `ETag` and `PartNumber`
4. Complete upload by calling `complete_multipart_upload` with all the parts with `{'Etag', 'PartNumber'}`

Reply:
I see..this makes sense. Thanks for the detailed explanation.